### ISSUE TITLE
Installing error in ubuntu

### ISSUE THREAD

**Issue by NishizawaKazuhiro** [velocyto-team/velocyto.R#39]
Hi, now I tried to install velocyto. But when I run install_github("velocyto-team/velocyto.R"), my PC showed this error below.

Doesn' find /usr/bin/ld: -lboost_filesystem 
Doesn' find /usr/bin/ld: -lboost_system
collect2: error: ld returned 1 exit status
/usr/local/lib/R/share/make/shlib.mk:6: fail the  'velocyto.R.so' 
make: *** [velocyto.R.so] error 1
ERROR: compilation failed for package ‘velocyto.R’
* removing ‘/usr/local/lib/R/site-library/velocyto.R’
 run(bin, args = real_cmdargs, stdout_line_callback = real_callback(stdout), error: 
  System command error

My PC environment is Ubuntu16.04(x86_64-pc-linux-gnu arch x86_64),R version is  3.4.1 (2017-06-30).

If you have some idea,please teach me.
Thank you.

**Comment by NishizawaKazuhiro** [comment]
Yes,I installed this software. But there is still the error message.

**Comment by raph06** [comment]
Got the same error,

> Doesn' find /usr/bin/ld: -lboost_filesystem
Doesn' find /usr/bin/ld: -lboost_system

 solved by the following command:

` sudo apt-get install libboost-all-dev`

**Comment by NishizawaKazuhiro** [comment]
>raph06

Thank you!
I could install it too!!
